tests/foreman/api/test_repository.py

```diff
@@ -2532,55 +2414,90 @@ def test_positive_create_with_long_token(self, repo_options, repo):
         :expectedresults: repo from registry with long password can be created
          and synced
         """
-        # First we want to confirm the provided token is > 255 charaters
-        assert (
-            len(repo_options['upstream_password']) > 255
-        ), 'Use a longer (>255) token for long_pass_test_registry'
+        # Make sure there is a long token repo to sync with
+        container_repos = [
+            repo
+            for _, repo in settings.container_repo.registries.items()
+            if getattr(repo, 'long_pass', False)
+        ]
+        try:
+            container_repo = container_repos[0]
+        except IndexError:
+            pytest.skip('No registries with "long_pass" set to true')
+        for docker_repo_name in container_repo.repos_to_sync:
+            repo_options = dict(
+                content_type='docker',
+                docker_upstream_name=docker_repo_name,
+                name=gen_string('alpha'),
+                upstream_username=container_repo.username,
+                upstream_password=container_repo.password,
+                url=container_repo.url,
+            )
+            repo_options['organization'] = module_org
+            repo_options['product'] = module_product
 
-        for k in 'name', 'docker_upstream_name', 'content_type', 'upstream_username':
-            assert getattr(repo, k) == repo_options[k]
-        repo.sync()
-        assert repo.read().content_counts['docker_manifest'] > 1
+            # First we want to confirm the provided token is > 255 characters
+            if not len(repo_options['upstream_password']) > 255:
+                pytest.skip('The "long_pass" registry does not meet length requirement')
+
+            repo = entities.Repository(**repo_options).create()
+
+            @request.addfinalizer
+            def clean_repo():
```

Comment:
Original test docstring does not call for deletion of repo.

Reply:
Correct, I opted to add some cleanup. Is there some harm in doing cleanup for these tests? It takes less than a second to make the call.

Reply:
@mshriver time to make the call is not the issue, time for Satellite to process all the tasks from all the tests when everyone  decides to add tear-down steps to their tests, just because they can, could become an issue. The time spent in doing all this is also a factor.

Reply:
I think that this is an incredibly minimal load, especially considering we're only running one test at a time per satellite target in our CI.  There aren't any other tests trying to schedule any other tasks on the satellite under test while these deletes are called in teardown.  If Satellite can't handle that amount of load, we've got some performance/usability bugs to start writing.

Reply:
the code is already in place and I don't care either way since the impact is minimal on both sides.